Update test_setup_environment.py to pass on local machines and not run in GitHub Actions

LukeParky opened the pull request:
To test enter the following commands from project root:
1. Start database:
 `docker-compose up -d db_postgres`
3. Run test_setup_environment.py:   
`python -m tests.test_setup_environment.test_setup_environment`
4. Run all tests: 
`pytest`

DESCRIPTION OF PR:

This test was originally written to teach some testing skills and as an incentive to fix some errors.
The errors were fixed in #81 but the tests still needed to pass.

Because these tests integrate with the database it does not match with our current testing platform.
However, they can run locally.

I have decided to allow these tests to run locally on Windows but not on docker or in the automated test runner. Please give me your thoughts on this.

## Developer Checklist
- [X] Make code change
- [X] Update tests
  - [X] Update / create new tests
  - [X] Ensure these tests have the expected behaviour
  - [X] Test locally and ensure tests are passing
- [X] Update documentation
~~- [ ] Readme~~
  - [X] Docstrings
  - [X] Comments
~~- [ ] Wiki~~

## Reviewer Checklist
- [ ] Check new code for code smells
- [ ] Check new tests
  - [ ] Ensure adequate coverage
  - [ ] Check for code smells within tests
- [ ] Check if documentation needs updating
  - [ ] Readme
  - [ ] Docstrings
  - [ ] Comments
  - [ ] Wiki

caseyli92 commented on tests/test_digitaltwin/test_setup_environment.py:
```diff
@@ -1,28 +1,38 @@
 
-    @unittest.skip("Skipping until we work on https://github.com/GeospatialResearch/Digital-Twins/issues/23")
+    @pytest.mark.skipif(
+        platform.system() != "Windows",
```
What if the developer uses other operating systems, such as Linux, for local development? Same goes for the one below.

LukeParky replied:
Casey and I are having a discussion about implementing a environment variable to be able to switch off these types of tests that require the database, so that they can be avoided only on the test runner or other special cases as needed. I think this could be a good idea, but am wary of having developers turn it off for convenience and never turning it back on. I will implement it for review

caseyli92 reviewed (commented):
Just a bit unsure about the part where we check if the platform is not Windows. I remember you mentioned setting up VMs for developers, and I assume they will be using Linux VMs, but it's not a major concern anyway.

LukeParky commented:
Hi @caseyli92 I addressed your concerns and have pushed my changes to allow the use of an environment variable. Could you have another review, thanks.

caseyli92 reviewed (approved):
Looks good :) The environment variable is only used in testing the `setup_environment.py` anyway, and as long as it is always set to True (default) in their `.env` file, it should be fine.
